easybuild/easyconfigs/d/dill/dill-0.3.4-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
```

Comment:
Previous version `dill-0.3.4-GCCcore-10.3.0.eb` has an attribution in the header. Please keep it in this update.

Reply:
@lexming fixed in e486f98357